doc/howto/multifederation.md

```diff
@@ -6,82 +6,93 @@ This document provides an initial step by step description of the technical solu
 
 The description of the steps is based on the diagram that Steffen draw in the board.
 
+
 ## Entities
-* `RP`. A Relying Party.
-* `OP`. A OpenID Provider.
-* `SS_FA`. The Signing Service for Federation A (`FA`).
-* `SS_FB`. The Signing Service for Federation B (`FB`).
-* `SS_FE`. The Signing Service for the EduGAIN federation (`FE`).
+* `RP1`. A Relying Party.
+* `OP1`. A OpenID Provider.
+* `MSS_FA`. The Metadata Signing Service for Federation A (`FA`).
+* `MSS_FB`. The Metadata Signing Service for Federation B (`FB`).
+* `MSS_CE`. The Metadata Signing Service for the community of federations E (`CE`). This might be eduGAIN-OIDC..
+
+
+## Remarks
+
+* A `RP`/`OP` that talks to `MSS_FA` is called `member of federation A`
+* A `RP`/`OP` might be part of multiple federations when talking to multiple MSS.
+* A `RP`/`OP` might be part of multiple communities. It might even be part of the same community multiple times via membership in several federations.
+* No revocation mechanism for either keys and/or signed MS is considered. Instead revocations should be handled by short-lived signatures, i.e. 15 minutes.
+
 
 ## The Signing Service
 The Signing Service is a key part of the architecture. Each federation MUST deploy one.
-A Signing Service (SS) has control over the private part of the Federation Key. Entities (RP/OP/SS) can enrol to become part of the federation. The SS gets access to the public parts of the signing_key of the enrolled entities and use them to generate signed MS for them.
+A Metadata Signing Service (MSS) has control over the private part of the Federation Key. Entities (RP1/OP1/MSS) can enrol to become part of the federation. The MSS gets access to the public parts of the signing_key of the enrolled entities and use them to generate signed Metadata Statement (`MS`) for them.
+
```

Comment:
Should we call it SMS? Roland has the term "sms" in many parts of the code and configuration.

Reply:
Fine for me.

Reply:
Maybe in the future, I want everyone agreeing on this first.